addons/mail/models/res_partner.py

```diff
@@ -18,10 +18,9 @@ class Partner(models.Model):
     """ Update partner to add a field about notification preferences. Add a generic opt-out field that can be used
        to restrict usage of automatic email templates. """
     _name = "res.partner"
-    _inherit = ['res.partner', 'mail.thread', 'mail.activity.mixin', 'mail.blacklist.mixin']
+    _inherit = ['res.partner', 'mail.activity.mixin', 'mail.blacklist.mixin', 'mail.thread']
```

Comment:
if mail.blacklist.mixin inherits from mail.thread, no need to inherit from mail.thread a second time

Reply:
If mail.blacklist.mixin inherit from mail_thread, we should apply the same logic as for mail.thread.cc, it isn't a real mixin, it makes sense to inherit from mail_thread since mail_thread methods are used or override, but a rename could be interesting to show that we don't need to add a specific inherit to mail.thread, something like mail.thread.blacklist.

Reply:
I agree. mail.thread.blacklist seems fine.